validation/pipeline/qase/schemaupload/main.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strconv"
+	"strings"
+
+	"github.com/rancher/tests/validation/pipeline/qase"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	_, callerFilePath, _, _ = runtime.Caller(0)
+	basepath                = filepath.Join(filepath.Dir(callerFilePath), "..", "..", "..", "..")
```

Comment:
would be more in favor of a const `../../../..` and calling the filepath.Join in functions instead of saving it as a var.

Reply:
I need to get the root directory path a few times and this just made it easier instead of redoing the logic each time